python/graphstorm/gconstruct/transform.py

```diff
@@ -1810,9 +1834,12 @@ def process_labels(data, label_processors):
     -------
     dict of tensors : labels (optional) and train/validation/test masks.
     """
-    print(label_processors)
-    assert len(label_processors) == 1, "We only support one label per node/edge type."
-    return label_processors[0](data)
+    assert len(label_processors) >= 1, \
+        "Number of label_processors must be one or more."
+    ret = {}
+    for label_processor in label_processors:
+        ret.update(label_processor(data))
```

Comment:
What about adding printing for the label_processors back here?

Reply:
What do you mean by printing?
Adding a debug info?

Reply:
Yep. I see we originally have print(label_processors)

Reply:
Added a logging.debug